packages/tangle/tipmanager.go

```diff
@@ -363,8 +363,7 @@ func (t *TipManager) Tips(p payload.Payload, countParents int) (parents MessageI
 }
 
 func (t *TipManager) isPastConeTimestampCorrect(messageID MessageID) (timestampValid bool) {
-	now := clock.SyncedTime()
-	minSupportedTimestamp := now.Add(-t.tangle.Options.TimeSinceConfirmationThreshold)
+	minSupportedTimestamp := t.tangle.TimeManager.Time().Add(-t.tangle.Options.TimeSinceConfirmationThreshold)
```

Comment:
With the new time types (CTT, RCTT, FTT, RFTT) added to the TimeManager, you might want to change that condition.